specs/protocol/rollup-node-p2p.md

```diff
@@ -277,6 +282,11 @@ A block is structured as the concatenation of:
   - `signature`: A `secp256k1` signature, always 65 bytes, `r (uint256), s (uint256), y_parity (uint8)`
   - `parentBeaconBlockRoot`: L1 origin parent beacon block root, always 32 bytes
   - `payload`: A SSZ-encoded `ExecutionPayload`, always the remaining bytes.
+- V4 topic
+  - `signature`: A `secp256k1` signature, always 65 bytes, `r (uint256), s (uint256), y_parity (uint8)`
+  - `parentBeaconBlockRoot`: L1 origin parent beacon block root, always 32 bytes
```

Comment:
Question: Is my assumption correct that this is an additive list?

Reply:
the block header RLP list? Kind of, attributes have been reused / changed meaning in the past, but it's meant to be backward compatible.
The SSZ version that is used for Gossip, with the `parentBeaconBlockRoot` added to it, is already non-standard, and not strictly speaking SSZ anymore (due to a missing offset value).